cmake/OpenSimConfig.cmake.in

```diff
@@ -75,3 +77,7 @@ set(@CMAKE_PROJECT_NAME@_JAVA_FILES_DIR
 set(@CMAKE_PROJECT_NAME@_JAR_FILE 
     @PACKAGE_CMAKE_INSTALL_PREFIX@/@OPENSIM_INSTALL_JAVAJARDIR@/@SWIG_JAVA_JAR_NAME@)
 
+find_package(Simbody ${SIMBODY_VERSION_TO_USE}
+    PATHS "@SIMBODY_HOME@" NO_MODULE NO_DEFAULT_PATH)
```

Comment:
This is correct if `OPENSIM_COPY_SIMBODY` is OFF, but if `OPENSIM_COPY_SIMBODY` is ON, then we should be using the Simbody that is packaged with OpenSim.